app/src/marketplace/flowtype/store-state.js

```diff
@@ -46,7 +46,8 @@ export type MyProductListState = {
 
 // my purchases
 export type MyPurchaseListState = {
-    ids: ProductIdList,
+    products: ProductIdList,
+    subscriptions: ProductIdList,
```

Comment:
`products` contains the visible product ids (with the current filter applied), `subscriptions` are the fetched (unique) subscription ids.